code/src/css/moviedetails.css

```diff
@@ -0,0 +1,101 @@
+.main-container{
+    position: relative;
+}
+
+.main-container .background{
+    min-height: 100vh;
+    display: flex;
+    background-size: cover;
+    flex-direction: column;
+    justify-content: flex-end;
+}
+
+.details-container{
```

Comment:
Great to make the text more visible. Another way to achieve that could be expanding the gradient from line 37 in MovieDetails.